### PR TITLE
Fix correct staking calculation, add clarification tooltips

### PR DESCRIPTION
**Pull request by Siegrift**
Fixes https://github.com/api3dao/api3-dao-dashboard/issues/208

### CODE REVIEW

File: src/pages/dashboard/staking/staking-pool.tsx [hunk 1]
```diff
@@ -5,57 +5,67 @@ import RadialChart from './radial-chart';
...
   const stakedPercentage = stakingPool ? Number.parseFloat(round(stakingPool.stakedPercentage)) : undefined;
   const totalStaked = data ? formatAndRoundApi3(data.totalStake) : UNKNOWN_NUMBER;
   const stakingTargetInTokens = stakingPool ? formatAndRoundApi3(stakingPool.stakingTargetInTokens) : UNKNOWN_NUMBER;
 
   const currentApyText = currentApy === UNKNOWN_NUMBER ? currentApy : `${currentApy}%`;
-  const annualInflationRateText =
-    annualInflationRate === UNKNOWN_NUMBER ? annualInflationRate : `${annualInflationRate}%`;
+  const annualTotalSupplyGrowthText =
+    annualTotalSupplyGrowth === UNKNOWN_NUMBER ? annualTotalSupplyGrowth : `${annualTotalSupplyGrowth}%`;
+  const estimatedValueTooltip = 'Value is based on the current APR and will change in time';
```
Comment (Siegrift) [inline src/pages/dashboard/staking/staking-pool.tsx:24]: @bbenligiray Not sure if you meant the message in the issue "Value is extrapolated based on the current APR" literally, but I dislike the "extrapolated" word as it sounds too technical.

However, I don't have strong preference about what should the tooltip say, so feel free to insist on something.

**Review by Siegrift — COMMENTED** [review]
@karolko @andreogle I've made some changes to the CSS as it looked a bit too complex and didn't look that good on other resolutions.

I've tried to be aligned with the designs in Figma. Please check out if I missed something.

File: src/contracts/helpers.test.ts [hunk 2]
```diff
@@ -32,11 +32,11 @@ describe('calculateAnnualInflationRate and friends', () => {
       expect(formatApi3(calculateAnnualMintedTokens(totalStake, apy))).toBe('55,284,457.03');
     });
 
-    test('calculateAnnualInflationRate', () => {
+    test('calculateAnnualTotalSupplyGrowth', () => {
       const apy = calculateApy(currentApr);
       const mintedTokens = calculateAnnualMintedTokens(totalStake, apy);
 
-      expect(calculateAnnualInflationRate(mintedTokens, API3_TOKEN_SUPPLY)).toBe(35.602054);
+      expect(calculateAnnualTotalSupplyGrowth(mintedTokens, API3_TOKEN_SUPPLY)).toBe(55.284456999999996);
```
Comment (Siegrift) [inline src/contracts/helpers.test.ts:39]: Does the numbers in the "corrected" tests make sense?

File: src/pages/dashboard/staking/staking-pool.tsx (hunk 1 above)
Comment (bbenligiray) [reply src/pages/dashboard/staking/staking-pool.tsx:24]: This is slightly better: `Value is based on the current APR and may change over time`

File: src/contracts/helpers.test.ts (hunk 2 above)
Comment (bbenligiray) [reply src/contracts/helpers.test.ts:39]: Yeah

**Review by karolko — APPROVED** [review]

**Review by andreogle — APPROVED** [review]
I'm unable to get the project running locally at the moment to check the CSS changes, but the PR looks good overall

**Pull request merged**